libs/eo/core/globalization/assets-localization/i18n/en.ts

```diff
@@ -238,6 +238,10 @@ export const EN_TRANSLATIONS: TranslationKeys = {
     logout: '{{shared.logout}}',
   },
   shared: {
+    notWhitelistedError: {
+      title: 'No access',
+      message: `You do not yet have access to issue Granular Guarantees of Origin.<br /> Contact support at <a href="tel:+4570222810">+45 70 22 28 10</a> or <a href="mailto:datahub@energinet.dk?cc=mil@energinet.dk">datahub@energinet.dk</a>.`,
```

Comment:
nitpick: to match the danish wording Micahel suggested, perhaps it should be:
```suggestion
      message: `You do not have access to issue Granular Guarantees of Origin.<br /> Contact support at <a href="tel:+4570222810">+45 70 22 28 10</a> or <a href="mailto:datahub@energinet.dk?cc=mil@energinet.dk">datahub@energinet.dk</a>.`,
```
Without the "yet"